app/routes/circulars/circulars.lib.ts

```diff
@@ -143,3 +147,27 @@ export function formatAuthor({
   else if (!affiliation) return `${name} <${email}>`
   else return `${name} at ${affiliation} <${email}>`
 }
+
+export function parseEventFromSubject(value: string) {
+  const subjectMatchers: SubjectMatcher[] = [
+    [/GRB[.\s|_|-]*(\d{6}[a-zA-Z|.]\d*)/, ([, id]) => `GRB ${id}`],
```

Comment:
In the delimiter part of the pattern, `[.\s|_|-]*`, does the `.` match any character? Or does it match a literal period?

Also: how about we do case-insensitive matching with the regexp `i` flag? That will make the parser a bit more forgiving, and also will eliminate the need to have `[a-zA-Z]`.